gettsim/demographic_vars.py

```diff
@@ -9,7 +9,9 @@
 from gettsim.typing import IntSeries
 
 
-def anz_minderj_hh(hh_id: IntSeries, alter: IntSeries, kind: BoolSeries) -> IntSeries:
+def anz_kinder_bis_18_hh(
```

Comment:
Should this not be `anz_kinder_bis_17_hh` ? Apologies if I was responsible for that. If we write `bis_X`, it should be interpreted as `<=`.